File: src/main/java/org/jdc/template/model/database/main/individualdata/IndividualData.java
/*
 * IndividualData.java
 *
 * Created: 11/20/2016 10:09:03
 */



package org.jdc.template.model.database.main.individualdata;

import android.database.Cursor;
import org.dbtools.android.domain.database.contentvalues.DBToolsContentValues;


public class IndividualData extends IndividualDataBaseRecord {


    public IndividualData(IndividualData record) {
        super(record);
    }

    public IndividualData(Cursor cursor) {
        setContent(cursor);
    }

    public IndividualData(DBToolsContentValues values) {
        setContent(values);
    }

    public IndividualData() {
    }


}

File: src/main/kotlin/org/jdc/template/model/database/main/individualdata/IndividualData.kt
/*
 * IndividualData.kt
 *
 * Created: 11/20/2016 08:38:17
 */



package org.jdc.template.model.database.main.individualdata

import android.database.Cursor
import org.dbtools.android.domain.database.contentvalues.DBToolsContentValues


class IndividualData : IndividualDataBaseRecord {


    constructor() {
    }

    constructor(record: IndividualData) : super(record) {
    }

    constructor(cursor: Cursor) {
        setContent(cursor)
    }

    constructor(values: DBToolsContentValues<*>) {
        setContent(values)
    }


}